build.sbt

```diff
@@ -111,7 +111,10 @@ lazy val maml = crossProject.in(file("."))
       geotrellis("layer").value,
       geotrellis("proj4").value
     )
-  ).jsSettings(name := "maml-js")
+  ).jsSettings(
+    name := "maml-js",
+    libraryDependencies += geotrellis("raster").value
```

Comment:
I'm wondering, would it work in production? That was required since I added TargetCell codecs. /cc @moradology